### ISSUE TITLE
[BUG] I can press the confirm button and incorrect button at the same time and they count

### ISSUE THREAD

**Issue by houssam1122** [BizTech-Morocco/mosa3ada.biztech.ma#83]
I can press the confirm button and incorrect button at the same time and they count :

https://github.com/BizTech-Morocco/mosa3ada.biztech.ma/assets/111252969/7b09b5be-931e-4d6b-a4da-2c6980fb7f35

**Good Luck !**

**Comment by Jineshbansal** [comment]
@soufyanelf Can I work on this issue?

**Comment by soufyanelf** [comment]
@Jineshbansal sure. Thanks for the joining Mosa3ada

**Comment by Jineshbansal** [comment]
@soufyanelf Can you help me in testing my code?

**Comment by Kathuria** [comment]
@Jineshbansal : You can block clickable feature of button until you have updated response. For UI experience, you can show button i.e. Correct/Incorrect in greyed out visual as soon as clicked i.e. not allowed to click until response received. This will be better as we don't want user to click twice and we can increment count as soon as we receive with loading shimmer.


Rest @houssam1122 or @soufyanelf, Can opine.

**Comment by soufyanelf** [comment]
> @Jineshbansal : You can block clickable feature of button until you have updated response. For UI experience, you can show button i.e. Correct/Incorrect in greyed out visual as soon as clicked i.e. not allowed to click until response received. This will be better as we don't want user to click twice and we can increment count as soon as we receive with loading shimmer.
> 
> Rest @houssam1122 or @soufyanelf, Can opine.

the user need to click only on one of them not both. when user click on one the other should become disabled.

**Comment by merouaneamqor** [comment]
@soufyanelf I can work On that if needed 
I have a solution on mind and it can be an easy win

**Comment by soufyanelf** [comment]
Yes sure but please check staging.mosa3ada.ma just to make sure is not already done

**Comment by Jineshbansal** [comment]
Sorry, I am unable to solve this issue @soufyanelf can you please remove me from assignees
